### PR TITLE
PROJ21-1221 Sync enrollment targets from Google Sheet to Progress Mon…

### PR DESCRIPTION
**Pull request by susanxiong**
…itoring

Using pygsheets to automatically pull the enrollment targets into `custom.SchoolMint_ProgressMonitoring` table.

I'm going to add the docs as a separate PR since there are so many files.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -185,6 +186,24 @@ def process_fact_daily_status(conn):
...
 
+def sync_enrollment_targets(conn, school_year):
+    """
+    Pull enrollment target numbers from spreadsheet and write to ProgressMonitoring table.
+    """
+    client = pygsheets.authorize(service_file="service.json")
+    sheet = client.open_by_key(os.getenv("TARGETS_SHEET_ID"))
+    worksheet = sheet.worksheet_by_title(os.getenv("TARGETS_SHEET_TITLE"))
+    df = worksheet.get_as_df()
+    conn.engine.execute(  # drop into sqlalchemy because we are locked into an older version of sqlsorcery
```
Comment (dchess) [inline main.py:197]: Are you able to use SQLSorcery like this?

```python
table = sql.table("SchoolMint_ProgressMonitoring")
table.delete().where(table.c.SchoolYear4digit == school_year)
```

File: main.py [hunk 2]
```diff
@@ -205,6 +224,8 @@ def main():
         process_change_tracking(conn)
         process_fact_daily_status(conn)
 
+        sync_enrollment_targets(conn, school_year)
```
Comment (dchess) [inline main.py:227]: Can we add arg parse so this is optional at runtime?

File: main.py (hunk 1 above)
Comment (susanxiong) [reply main.py:197]: nope  - "'MSSQL' object has no attribute 'table'"

Comment (dchess) [reply main.py:197]: Bummer. What version of sqlsorcery is this on?

Comment (susanxiong) [reply main.py:197]: 0.1.2

File: main.py (hunk 2 above)
Comment (susanxiong) [reply main.py]: added!

File: main.py (hunk 1 above)
Comment (dchess) [reply main.py:200]: Ah. That version is pretty barebones.

**Review by dchess — APPROVED** [review]

**Pull request merged**